static/js/HeaderAutocomplete.jsx

```diff
@@ -466,34 +452,46 @@ const SuggestionsGroup = ({ suggestions, initialIndexForGroup, getItemProps, hig
     onNavigate && onNavigate();
   }
 
+  const renderInput = (highlightedIndex, highlightedSuggestion, getInputProps, setInputValue)=> {
 
-  return (
-    <div className={"search-container"}>
-      <SearchInputBox
+      return(
+            <SearchInputBox
             getInputProps={getInputProps}
-            suggestions={suggestions}
+            highlightedSuggestion={highlightedSuggestion}
             hideHebrewKeyboard={hideHebrewKeyboard}
             highlightedIndex={highlightedIndex}
             setInputValue={setInputValue}
 
             setSearchFocused={setSearchFocused}
             searchFocused={searchFocused}
 
-            submitSearch={submitSearch}
-            redirectToObject={redirectToObject}
+            submitSearch={submitSearch.bind(null, getInputProps().onChange)}
+            redirectToObject={redirectToObject.bind(null, getInputProps().onChange)}
       />
-      <div
-        {...getMenuProps()}
-        className={"autocomplete-dropdown"}
-      >
-      {/*//debug: make following condition always truthy:*/}
-          {(isOpen && searchFocused) &&
-              <SuggestionsDispatcher suggestions={suggestions} getItemProps={getItemProps} highlightedIndex={highlightedIndex}
-                   getInputProps={getInputProps} submitSearch={submitSearch} redirectToObject={redirectToObject}
+      )
+  };
+
+  const renderItems =(suggestions, highlightedIndex, getItemProps, getInputProps) => {
+
+      return(
+             <SuggestionsDispatcher suggestions={suggestions} getItemProps={getItemProps} highlightedIndex={highlightedIndex}
+                   getInputProps={getInputProps}
+                                    submitSearch={submitSearch.bind(null, getInputProps().onChange)}
+                                    redirectToObject={redirectToObject}
               />
-          }
-      </div>
-    </div>
+      )
+  };
+
+
+  return (
+      <GeneralAutocomplete
+          containerClassString='search-container'
+          dropdownMenuClassString='autocomplete-dropdown'
+          renderInput={renderInput}
+          renderItems={renderItems}
+          getSuggestions={fetchSuggestions}
+          shouldDisplaySuggestions={(isOpen)=> {return isOpen && searchFocused}}
+      />
   );
 };
-export {Autocomplete};
\ No newline at end of file
+export {HeaderAutocomplete};
```

Comment:
move export to class definition of HeaderAutocomplete. `export const HeaderAutocomplete = ...`